packages/shared/src/utils/constants/contract.ts

```diff
@@ -262,44 +262,53 @@ export const unsupported_contract_types_list = [
 export const getCardLabels = () =>
     ({
         APPLY: localize('Apply'),
-        BARRIER: localize('Barrier:'),
-        BUY_PRICE: localize('Buy price:'),
+        BARRIER: localize('Barrier'),
```

Comment:
Not sure if we should change this directly in here since this object is still being used in Trader V1

Reply:
that's true, I will fix the old uses

Reply:
This should be fixed now